docs/examples/java-examples.md

```diff
@@ -0,0 +1,393 @@
+# Java GraphSDK Example
+
+## Setting up package dependencies
+
+The GraphSDK JAR package is hosted in a GitHub Maven repository. The URL for the Maven repo is:
+https://maven.pkg.github.com/LibertyDSNP/graph-sdk
+
+and the package is: `io.amplica.graphsdk`
+
+## Prerequisites
+
+Since GraphSDK does not itself interact with the blockchain, for the following examples, we'll make use of a proposed class, `FrequencyService`, that provides an API to Frequency blockchain RPCs and Extrinsics. The structure/syntax for your own blockchain interface may be somewhat different.
+
+## Basic Social Graph Workflow
+
+The following code examples illustrate how to implement various steps in the workflows represented [here](basic-flow.md)
+
+## Code Example
+
+```kotlin
+  fun addNewGraphKeyOrReadPublicKeysWorkflow() {
+    // Graph keys are x25519 and different from msa control keys which are sr25519 and should not get mixed
+    val newKeyPair = Graph.generateKeyPair(GraphKeyType.X25519)
+    val msaId = 1L;
+    val publicKey = newKeyPair.publicKey
+
+    // ----- SETUP GRAPH ----- //
+    // initialize the Graph in memory
+    Configuration configuration = Configuration.getMainnet()
+    val graph = Graph(configuration)
+
+    // Get the schema ID for the graph keys
+    val schemaId = configuration
+    .getEnvironment()
+    .getConfig()
```

Comment:
Is this right? Feels off to be calling .getConfig on a variable of type Configuration